reference/api/tasks.md

```diff
@@ -0,0 +1,192 @@
+# Tasks
+
+The `tasks` route gives information about the progress of [asynchronous operations](/learn/advanced/asynchronous_operations.md).
+
+:::note
+The task `uid` is incremented **globally.**
+:::
+
+## Get all tasks
+
+<RouteHighlighter method="GET" route="/tasks"/>
+
+List all tasks globally regardless of the index involved. The `task` objects are contained in the `results` array.
+
+### Example
+
+<CodeSamples id="get_all_tasks_1" />
+
+#### Response: `200 Ok`
+
+```json
+{
+    "results": [
+        {
+            "uid": 1,
+            "indexUid": "movies_reviews",
+            "status": "enqueued",
+            "type": "documentsAddition",
+            "duration": null,
+            "enqueuedAt": "2021-08-12T10:00:00.000000Z",
+            "startedAt": null,
+            "finishedAt": null
+        },
+        {
+            "uid": 0,
```

Comment:
In the future, we should probably try to make the 0th task always an index creation—doesn't make much sense otherwise. Not a big deal though.